libunwind/src/UnwindRustSgxSnprintf.c

```diff
@@ -0,0 +1,867 @@
+/*
```

Comment:
I think this entire file is no longer necessary?

Reply:
correct, removed, thank you